content/en/security/application_security/enabling/serverless.md

```diff
@@ -216,9 +218,782 @@ To install and configure the Datadog Serverless Framework plugin:
+ADD https://dtdg.co/latest-java-tracer /dd_tracer/java/dd-java-agent.jar
+ENV DD_SERVICE=datadog-demo-run-java
+ENV DD_ENV=datadog-demo
+ENV DD_VERSION=1
+ENV DD_APPSEC_ENABLED=1
+ENTRYPOINT ["/app/datadog-init"]
+CMD ["./mvnw", "spring-boot:run"]
+```
+#### Explanation
+
+1. Copy the Datadog `serverless-init` into your Docker image.
+   ```dockerfile
+   COPY --from=datadog/serverless-init:1 /datadog-init /app/datadog-init
+   ```
+
+2. Add the Datadog Java tracer to your Docker image. 
+   ```dockerfile
+   ADD https://dtdg.co/latest-java-tracer /dd_tracer/java/dd-java-agent.jar
+   ```
+   If you install the Datadog tracer library directly in your application, as outlined in the [manual tracer instrumentation instructions][1], omit this step.
+
+3. (Optional) Add Datadog tags.
+   ```dockerfile
+   ENV DD_SERVICE=datadog-demo-run-java
+   ENV DD_ENV=datadog-demo
+   ENV DD_VERSION=1
+   ENV DD_APPSEC_ENABLED=1
+   ```
+
+4. Change the entrypoint to wrap your application in the Datadog `serverless-init` process. 
+   **Note**: If you already have an entrypoint defined inside your Dockerfile, see the [alternative configuration](#alt-java).
+   ```dockerfile
+   ENTRYPOINT ["/app/datadog-init"]
+   ```
+
+5. Execute your binary application wrapped in the entrypoint. Adapt this line to your needs.
+   ```dockerfile
+   CMD ["./mvnw", "spring-boot:run"]
+   ```
+
+#### Alternative configuration {#alt-java}
+If you already have an entrypoint defined inside your Dockerfile, you can instead modify the CMD argument.
+
+{{< highlight dockerfile "hl_lines=7" >}}
+COPY --from=datadog/serverless-init:1 /datadog-init /app/datadog-init
+ADD https://dtdg.co/latest-java-tracer /dd_tracer/java/dd-java-agent.jar
+ENV DD_SERVICE=datadog-demo-run-java
+ENV DD_ENV=datadog-demo
+ENV DD_VERSION=1
+ENV DD_APPSEC_ENABLED=1
+CMD ["/app/datadog-init", "./mvnw", "spring-boot:run"]
+{{< /highlight >}}
+
+If you require your entrypoint to be instrumented as well, you can swap your entrypoint and CMD arguments instead. For more information, see [How `serverless-init` works](#how-serverless-init-works).
+
+{{< highlight dockerfile "hl_lines=7-8" >}}
+COPY --from=datadog/serverless-init:1 /datadog-init /app/datadog-init
+ADD https://dtdg.co/latest-java-tracer /dd_tracer/java/dd-java-agent.jar
+ENV DD_SERVICE=datadog-demo-run-java
+ENV DD_ENV=datadog-demo
+ENV DD_VERSION=1
+ENV DD_APPSEC_ENABLED=1
+ENTRYPOINT ["/app/datadog-init"]
+CMD ["your_entrypoint.sh", "./mvnw", "spring-boot:run"]
+{{< /highlight >}}
+
+As long as your command to run is passed as an argument to `datadog-init`, you will receive full instrumentation.
+
+[1]: /tracing/trace_collection/dd_libraries/java/?tab=containers#instrument-your-application
+
+{{% /tab %}}
+{{% tab "Go" %}}
+
+[Manually install][1] the Go tracer before you deploy your application. Compile your go binary with the "appsec" tag enabled (`go build --tags "appsec" ...`). Add the following instructions and arguments to your Dockerfile:
```

Comment:
On the preview page the `[Manually install][1]` link is not rendered... Is this expected?

Reply:
Great catch, I just pushed a fix for this.